### PR TITLE
LRQA-82363 | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://liferay.atlassian.net/browse/LRQA-82363

Tested on https://github.com/catarineaguiar/liferay-portal/pull/121

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/deployment/LPKG.testcase [hunk 1]
```diff
@@ -50,7 +51,12 @@ definition {
 
 		WaitForConsoleTextPresent(value1 = "The portal instance needs to be restarted to complete the installation");
 
-		User.firstLoginPG();
+		Navigator.openSpecificURL(url = "http://localhost:8080");
```
Comment (kyle-miho) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/deployment/LPKG.testcase:54]: This gets called first thing in firstLoginUI, no need for it

Comment (kyle-miho) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/deployment/LPKG.testcase:54]: @catarineaguiar fyi, ill make the change and forward

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/141208

**Pull request closed without merging**